.github/scripts/publish-rhel.sh

```diff
@@ -17,16 +17,17 @@ get_image()
         echo "Need first parameter as 'published' or 'not_published'." ; return 1
     fi
 
-    local FILTER="filter=deleted==false;${PUBLISHED_FILTER}"
-    local INCLUDE="include=total,data.repositories.tags.name,data.certified,data.container_grades,data._id"
+    local FILTER="filter=deleted==false;${PUBLISHED_FILTER};repositories.tags=em=(name=='${VERSION}')"
+    local INCLUDE="include=total,data.repositories.tags.name,data.certified,data.container_grades,data._id,data.creation_date"
+    local SORT_BY='sort_by=creation_date\[desc\]'
 
     local RESPONSE=$( \
         curl --silent \
              --request GET \
              --header "X-API-KEY: ${RHEL_API_KEY}" \
-             "https://catalog.redhat.com/api/containers/v1/projects/certification/id/${RHEL_PROJECT_ID}/images?${FILTER}&${INCLUDE}")
+             "https://catalog.redhat.com/api/containers/v1/projects/certification/id/${RHEL_PROJECT_ID}/images?${FILTER}&${INCLUDE}&${SORT_BY}")
 
-    echo "${RESPONSE}" | jq ".data[] | select(.repositories[].tags[]?.name==\"${VERSION}\")" | jq -s '.[0] | select( . != null)' | jq -s '{data:., total: length}'
+    echo "${RESPONSE}"
```

Comment:
This is now returning entire output. Later I see 

```
    local IS_PUBLISHED=$(get_image published "${RHEL_PROJECT_ID}" "${VERSION}" "${RHEL_API_KEY}" | jq -r '.total')
    if [[ $IS_PUBLISHED == "1" ]]; then
```
Is checking for `"1"` still valid? 
I am sure it is but just checking and may be a comment will be good

Reply:
Client side filtering was keeping the original structure from the server resposen :

```
{
  data:[...],
  total: <size>
}
```

Here's the sample response from the server:

```json

{
  "total": 1,
  "data": [
    {
      "repositories": [
        {
          "tags": [
            {
              "name": "5.3.8-jdk17"
            }
          ]
        }
      ],
      "certified": true,
      "container_grades": {
        "status": "completed",
        "status_message": "This image does not have any unapplied Critical or Important security errata"
      },
      "_id": "66c6d5e51cc0ec2e45e47c7b",
      "creation_date": "2024-08-22T06:08:37.423000+00:00",
      "_links": {
        "artifacts": {
          "href": "/v1/images/id/66c6d5e51cc0ec2e45e47c7b/artifacts"
        },
        "requests": {
          "href": "/v1/images/id/66c6d5e51cc0ec2e45e47c7b/requests"
        },
        "rpm_manifest": {
          "href": "/v1/images/id/66c6d5e51cc0ec2e45e47c7b/rpm-manifest"
        },
        "test_results": {
          "href": "/v1/images/id/66c6d5e51cc0ec2e45e47c7b/test-results"
        },
        "vulnerabilities": {
          "href": "/v1/images/id/66c6d5e51cc0ec2e45e47c7b/vulnerabilities"
        }
      }
    }
  ]
}

```

Reply:
got it. thanks